ignore.txt

```diff
@@ -6,3 +6,5 @@ mor
 te
 ond
 extraversion
+re-use
+continute
```

Comment:
what's this from? / where does `continute` exist in the code?

Reply:
it's in `janitor_util.ccp` script!

Reply:
Ah ok! Let's just update that comment's typo?